### DISCUSSION TITLE
IP属地显示效果修改建议

### DISCUSSION THREAD

**Discussion by ChaXxl** [the1812/Bilibili-Evolved#4622] [功能建议 / Ideas]
# 一、 问题
> ## 1. 这是 B站评论区开盒脚本的显示效果，日期和IP属地之间有空隙，看起来很舒服
> ![image](https://github.com/the1812/Bilibili-Evolved/assets/40326898/6fba14c8-a54f-4156-a5f0-6ceedfe57263)

<br/>

> ## 2. 但 Bilibili-Evolved 里面的显示效果很挤，看起来不舒服
> ![image](https://github.com/the1812/Bilibili-Evolved/assets/40326898/8515dc84-49b7-464f-b58d-4bb08c4dffb5)

<br/>
<br/>

# 二、 改进建议
## 增大日期和IP属地之间的间隔
![image](https://github.com/the1812/Bilibili-Evolved/assets/40326898/4e1f80fa-9575-4d61-a119-ce78027cc121)

**Comment by the1812** [comment]
#4506
